magento1-module/app/code/local/Divante/VueStorefrontBridge/controllers/PrzelewyController.php

```diff
@@ -0,0 +1,131 @@
+<?php
+// @codingStandardsIgnoreStart
+require_once 'AbstractController.php';
```

Comment:
use require_once using getModuleDir - for instance:
`require_once Mage::getModuleDir('controllers', 'Divante_VueStorefrontBridge') . DS . 'AbstractController.php';`

it's safer